src/compiler/objective_c_generator.cc

```diff
@@ -101,11 +101,13 @@ void PrintAdvancedSignature(Printer *printer,
 }
 
 inline map<string, string> GetMethodVars(const MethodDescriptor *method) {
```

Comment:
I reverted to the original after Craig's comment, but please let me know if you feel strongly about it.